crates/holochain/tests/tests/warrant_issuance.rs

```diff
@@ -74,7 +74,6 @@ async fn invalid_op_warrant_issuance_can_be_disabled() {
         .unwrap()[0]
         .test_read(move |txn| {
             let store = CascadeTxnWrapper::from(txn);
-            // TODO: The check_valid argument of get_warrants_for_agents should be removed once warrants are validated.
```

Comment:
This comment was wrong. The bool parameter must remain in the function `get_warrants_for_agent`, because authored warrants are not validated.